README.md

```diff
@@ -129,6 +129,38 @@ file { '/etc/tmpfiles.d/foo.conf':
 ~> Class['systemd::tmpfiles']
 ```
 
+### timer units
+Create a timer unit and a service unit to execute from
+that timer at the same time.
+
+```puppet
+systemd::timer{'runoften.timer':
```

Comment:
Should the timer be automatically added? It's a bit redundant.

Reply:
@ekohl I'm not sure what you mean here?

Reply:
Should it accept `systemd::timer{'runoften':` and in timer.pp add `.timer` internally?

Reply:
Got you, that `.timer` I went through this myself. I thought it was more consistent with `systemd::unit_file` for instance.

Certainly it would remove all that basename dancing.

Reply:
I think I prefer it with the .timer forced, it makes it consistent with
the `service_unit` parameter as well.